src/kerodon/impl.clj

```diff
@@ -86,12 +86,14 @@
     link
     (not-found "link" selector)))
 
-(defn- field-to-selector [elem]
-  (if-let [id (get-in elem [:attrs :id])]
-    (form-element-by {:id id})
-    (if (= "radio" (get-in elem [:attrs :type]))
-      (form-element-by (select-keys (:attrs elem) [:name :value]))
-      (form-element-by {:name (get-in elem [:attrs :name])}))))
+(defn- field-to-selector [elem & [selector]]
+  (if selector
```

Comment:
Good point.  In that case, this would be sufficient:

```clojure
(defn- form-element-selector [doc selector elem]
  (if (= :label (:tag elem))
    (label-to-selector doc elem)
    [selector]))
```

I noticed that in `form-element-by`, a `:form` tag is put at the head of the selector vector.  Is that necessary here?